kivymd/uix/button/button.py

```diff
@@ -874,27 +893,26 @@ def set_text_color(self, *args) -> NoReturn:
             default_icon_color = self.theme_cls.text_color
         self._icon_color = self.icon_color or default_icon_color
 
+    def on_icon_color(self, instance_button, color: list) -> NoReturn:
+        """If we are setting an icon color, set theme_icon_color to Custom.
+        For backwards compatibility (before theme_icon_color existed).
+        """
+        if color and (self.theme_text_color == 'Custom'):
```

Comment:
"""
        If we are setting an icon color, set theme_icon_color to Custom.
        For backwards compatibility (before theme_icon_color existed).
        """

        if color and (self.theme_text_color == 'Custom'):

Reply:
Done.